response_operations_ui/static/scss/components/load-button.scss

```diff
@@ -0,0 +1,340 @@
+#btn-load-sample, #btn-load-ci {
```

Comment:
Some of this css isn't being used right away by us in the html. Is our plan just to bring in everything relating to the components so we can use whatever going forward?

Reply:
Probably worth having a chat with @nicprice today. Was just being lazy last night to get it working